src/test/scala/com/microsoft/hyperspace/index/RefreshIndexTests.scala

```diff
@@ -208,6 +208,155 @@ class RefreshIndexTests extends QueryTest with HyperspaceSuite {
     }
   }
 
+  test(
+    "Validate refresh index (to handle deletes from the source data) " +
+      "for enforce delete on read.") {
+    // Save test data non-partitioned.
+    SampleData.save(
+      spark,
+      nonPartitionedDataPath,
```

Comment:
We should be able to utilize `withTempDir` here instead as well.

Reply:
Actually, we already had `withTempDir` under `HyperspaceSuite` and it served the purpose; I just switched the code to use that.